keps/sig-network/4559-kubelet-probes/README.md

```diff
@@ -213,35 +217,25 @@ UNRESOLVED section in the Summary.
       exec probe that uses `curl` or the like to connect to that IP
       from the pod itself.)
 
-    - If we implement this by creating a new set of probe/lifecycle handler
-      types, rather than by modifying the semantics of the existing probes,
-      then it should be possible to block the old probe types via Pod
-      Security admission, so that administrators can create
-      NetworkPolicies that don't include the kubelet probe hole.
-
 - Allow dual-stack / wrong-single-stack probing, either based on
   information in the probe definition (e.g. `ipFamily: IPv6` or
   `ipFamilyPolicy: RequireDualStack`) or by always doing "[Happy
   Eyeballs]" (probing IPv4 and IPv6 in parallel and only requiring one
   to succeed).
 
-- Make it easy for users who aren't doing "weird" things (eg,
-  readiness probe via LoadBalancer) to transition from the old probe
-  system to the new one.
-
-    - This might mean that they don't need to do anything, or it might
-      mean that there's an easy transition, like just changing
-      `tcpSocket:` to `tcp:` in their pod's probe definition.
-
 [`TCPSocketAction.Host`]: https://github.com/kubernetes/kubernetes/blob/v1.28.0/staging/src/k8s.io/api/core/v1/types.go#L2248
 [`HTTPGetAction.Host`]: https://github.com/kubernetes/kubernetes/blob/v1.28.0/staging/src/k8s.io/api/core/v1/types.go#L2219
 [Pod Security admission]: https://kubernetes.io/docs/concepts/security/pod-security-admission/
 [Happy Eyeballs]: https://tools.ietf.org/html/rfc8305
+[KEP #4940]: https://github.com/kubernetes/enhancements/pull/4942
 
 ### Non-Goals
 
 - Adding entirely-new probe types.
```

Comment:
Adding entirely new probe types that are not replacement for the existing probe types